### PR TITLE
Create a cluster pool ci-ocp-4-7-amd64-aws-us-east-1

### PR DESCRIPTION
**Pull request by hongkailiu**
https://issues.redhat.com/browse/DPTP-2097

- Use `ci-cluster-pool` project for the resources. According to Hive doc, pools should stay in a namespace as long as the bills for cloud come to the same team.
- Create the secrets used by the pool.

/cc @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

File: clusters/app.ci/ci-cluster-pool/ci-ocp-4.7.0-amd64-aws-us-east-1_clusterpool.yaml
Comment (stevekuznetsov) [inline clusters/app.ci/ci-cluster-pool/ci-ocp-4.7.0-amd64-aws-us-east-1_clusterpool.yaml:23]: We likely need this to be in it's own account, as otherwise it will impact our ability to manage quota with boskos

Comment (hongkailiu) [reply clusters/app.ci/ci-cluster-pool/ci-ocp-4.7.0-amd64-aws-us-east-1_clusterpool.yaml:23]: Not sure if you have seen `maxSize` (on top of `size`) in the cluster pool
https://github.com/openshift/hive/blob/73585aafdf3e43be77dc3eac29c3f27bede2a153/apis/hive/v1/clusterpool_types.go#L28

But yes, it will impact the quota as well.
So I should start to request a new account from DPP?

**Comment by hongkailiu** [comment]
The dryrun failed becasue
https://issues.redhat.com/browse/HIVE-1468

**Comment by stevekuznetsov** [comment]
This LGTM once it's in the new account

**Comment by hongkailiu** [comment]
Applying for a new account
https://issues.redhat.com/browse/DPP-7097

The failed job will pass next week (Fix is merged on the Hive and we have to wait for a weekly build of the operator).
Hive will be upgraded automatically.

**Comment by hongkailiu** [comment]
/hold

Testing with the new aws account.

**Comment by hongkailiu** [comment]
Tested locally, the new AWS account works.

```console
$ oc get clusterpool -n ci-cluster-pool
NAME                               READY   SIZE   BASEDOMAIN                  IMAGESET
ci-ocp-4-7-0-amd64-aws-us-east-1   1       1      hive.aws.ci.openshift.org   ocp-4.7.0-amd64

$ oc get cd --all-namespaces
NAMESPACE                                NAME                                     PLATFORM   REGION      CLUSTERTYPE   INSTALLED   INFRAID                       VERSION   POWERSTATE    AGE
ci-ocp-4-7-0-amd64-aws-us-east-1-lq78b   ci-ocp-4-7-0-amd64-aws-us-east-1-lq78b   aws        us-east-1                 true        ci-ocp-4-7-0-amd64-aw-qk95v   4.7.0     Hibernating   55m
```

/hold cancel

File: clusters/app.ci/ci-cluster-pool/ci-ocp-4-7-0-amd64-aws-us-east-1_clusterpool.yaml
Comment (alvaroaleman) [inline clusters/app.ci/ci-cluster-pool/ci-ocp-4-7-0-amd64-aws-us-east-1_clusterpool.yaml:8]: ftr, I think the z stream should not be in the label. Ppl don't care about that and its annoying to bump it every time a new pach release is published

Comment (hongkailiu) [reply clusters/app.ci/ci-cluster-pool/ci-ocp-4-7-0-amd64-aws-us-east-1_clusterpool.yaml]: Updated.

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**
